solr/core/src/test/org/apache/solr/cloud/OverseerCollectionConfigSetProcessorTest.java

```diff
@@ -563,6 +564,9 @@ private void handleCreateCollMessageProps(ZkNodeProps props) {
     try {
       if (CollectionParams.CollectionAction.CREATE.isEqual(props.getStr("operation"))) {
         String collName = props.getStr("name");
+        if (props.containsKey(CollectionAdminParams.COLL_CONF)) {
+          props.plus(ZkStateReader.CONFIGNAME_PROP, props.getProperties().remove("collection.configName"));
```

Comment:
"plus" returns a new ZkNodeProps without modifying it in-place!  Maybe you don't need this code any way? (I dunno)